src/WP_CLI/CommandWithUpgrade.php

```diff
@@ -825,7 +825,7 @@ private function parse_url_host_component( $url, $component ) {
 	 *
 	 * @return array{ name: string, url: string }|\WP_Error
 	 */
-	public function get_the_latest_github_version( $repo_slug ) {
+	private function get_the_latest_github_version( $repo_slug ) {
```

Comment:
These methods were public by default so they could be used somewhere else. So it would be good to set the visibility to `protected` instead of `private`.

Something like this has broken things in the past - https://github.com/wp-cli/wp-cli-tests/pull/170